packages/rest/test/unit/coercion/paramStringToBoolean.unit.ts

```diff
@@ -12,8 +13,68 @@ const BOOLEAN_PARAM = {
   schema: {type: 'boolean'},
 };
 
-describe('coerce param from string to boolean', () => {
-  test(BOOLEAN_PARAM, 'false', false);
-  test(BOOLEAN_PARAM, 'true', true);
-  test(BOOLEAN_PARAM, undefined, undefined);
+const REQUIRED_BOOLEAN_PARAM = {
```

Comment:
nitpick: `REQUIRED_BOOLEAN_PARAM_SPEC`

Reply:
I am trying to keep the name as short as possible 🙈 🙈